src/garage/experiment/local_tf_runner.py

```diff
@@ -340,73 +377,36 @@ def resume(self,
         assert self.train_args is not None, (
             'You must call restore() before resume().')
 
-        return self._train(
-            n_epochs=n_epochs or self.train_args.n_epochs,
-            n_epoch_cycles=n_epoch_cycles or self.train_args.n_epoch_cycles,
-            batch_size=batch_size or self.train_args.batch_size,
-            plot=plot or self.train_args.plot,
-            store_paths=store_paths or self.train_args.store_paths,
-            pause_for_plot=pause_for_plot or self.train_args.pause_for_plot,
-            start_epoch=self.train_args.start_epoch)
-
-    def _train(self,
-               n_epochs,
-               n_epoch_cycles,
-               batch_size,
-               plot,
-               store_paths,
-               pause_for_plot,
-               start_epoch=0):
+        if n_epochs:
+            self.train_args.n_epochs = n_epochs
+        if batch_size:
+            self.train_args.batch_size = batch_size
+        if n_epoch_cycles:
+            self.train_args.n_epoch_cycles = n_epoch_cycles
+        if plot:
+            self.train_args.plot = plot
+        if store_paths:
```

Comment:
This is a boolean, so you need to explicitly check that `store_paths is not None`.